traffic_ops/testing/api_contract/v4/test_cdns.py

```diff
@@ -0,0 +1,68 @@
+"""Api Contract Test Case for cdns endpoint"""
+import json
+import logging
+import pytest
+
+# Create and configure logger
+logger = logging.getLogger()
+
+
+@pytest.fixture(name="get_cdn_data")
+def get_cdn_prereq_data():
+    """PyTest Fixture to store prereq data for cdns endpoint"""
+    # Response keys for cdns endpoint
+    with open('prerequisite_data.json', encoding="utf-8", mode='r') as prereq_file:
+        data = json.load(prereq_file)
+    cdn_data = data["cdns"]
```

Comment:
quotation style should be consistent; linter won't force you to use one or the other, but for whatever it's worth I like to use double quotes because that's more familiar to people who use languages like Go and Java, where `"` and `'` aren't interchangeable.

Reply:
Used double quotes throughout all the files.